Content.Server/Administration/Logs/AdminLogManager.cs

```diff
@@ -316,6 +316,7 @@ private void Add(LogType type, LogImpact impact, string message, JsonDocument js
             Type = type,
             Impact = impact,
             Date = DateTime.UtcNow,
+            CurTime = _timing.CurTime.Ticks,
```

Comment:
Should this be stored relative to round start or something? Otherwise it'll be quite impractical to cross-reference this with replays.